docs/source/operations/conversions/unitconvert.rst

```diff
@@ -41,25 +41,29 @@ expected to be in units of decimalyears. This can be fixed with `unitconvert`::
 Parameters
 ################################################################################
 
-.. option:: +xy_in=<unit>
+.. option:: +xy_in=<unit> or <conversion_factor>
 
-    Horizontal input units. See :ref:`distance_units` for a list of available
-    units.
+    Horizontal input units. See :ref:`distance_units` and :ref:`angular_units`
+    for a list of available units. `<conversion_factor>` is the conversion factor
+    from the input unit to the International System unit.
```

Comment:
Perhaps just say that it is the conversion factor to meters? Otherwise I would say that "SI unit" is closer to standard nomenclature. Also valid below.

Reply:
"conversion factor to meters"... if it is linear unit, otherwise it is to radian. I can add that precision

Reply:
Done